### PR TITLE
translate the comments and some strings in the article of JavaScript functions

### PR DESCRIPTION
**Pull request by hanyujie2002**
…functions

### CODE REVIEW

**Comment by hanyujie2002** [comment]
人生第一次PR

File: files/zh-cn/learn/javascript/building_blocks/functions/index.md [hunk 1]
```diff
@@ -50,18 +50,18 @@ console.log(newString);
 var myArray = ['I', 'love', 'chocolate', 'frogs'];
 var madeAString = myArray.join(' ');
 console.log(madeAString);
-// the join() function takes an array, joins
-// all the array items together into a single
-// string, and returns this new string
+// join() 函数接受一个数组，将
+// 所有数组项连接成一个单一的
```
Comment (yin1999) [inline files/zh-cn/learn/javascript/building_blocks/functions/index.md:54]:
```suggestion
// 所有数组元素连接成一个单一的
```

File: files/zh-cn/learn/javascript/building_blocks/functions/index.md [hunk 2]
```diff
@@ -230,11 +230,11 @@ var newString = myText.replace('string', 'sausage');
 还应该注意，有时参数不是必须的 —— 您不必指定它们。如果没有，该功能一般会采用某种默认行为。作为示例，数组 [join（）](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Array/join)函数的参数是可选的：
 
 ```js
-var myArray = ['I', 'love', 'chocolate', 'frogs'];
+var myArray = ['我', '爱', '巧克力', '青蛙'];
 var madeAString = myArray.join(' ');
-// returns 'I love chocolate frogs'
+// returns '我 爱 巧克力 青蛙'
```
Comment (yin1999) [inline files/zh-cn/learn/javascript/building_blocks/functions/index.md:235]:
```suggestion
// 返回 '我 爱 巧克力 青蛙'
```

File: files/zh-cn/learn/javascript/building_blocks/functions/index.md [hunk 3]
```diff
@@ -230,11 +230,11 @@ var newString = myText.replace('string', 'sausage');
...
 var madeAString = myArray.join();
-// returns 'I,love,chocolate,frogs'
+// returns '我,爱,巧克力,青蛙'
```
Comment (yin1999) [inline files/zh-cn/learn/javascript/building_blocks/functions/index.md:237]: idto

**Review by yin1999 — APPROVED** [review]
Thanks. Welcome to the community

**Comment by schalkneethling** [comment]
Welcome to the community, @hanyujie2002  🎉 Thank you for your contribution. If you are interested in more ways to contribute to MDN Web Docs, please have a look at our [contributor task board](https://github.com/orgs/mdn/projects/25). If you have any questions or need help finding something to work on, please ask for help in our [community discussions](https://github.com/orgs/mdn/discussions). We look forward to more of your contributions. 🙌

**Pull request merged**